src/main/java/no/ntnu/mycbr/rest/AmalgamationFunctions.java

```diff
@@ -13,19 +12,22 @@
 
 public class AmalgamationFunctions {
```

Comment:
I wonder what is the purpose of this class. What it does is just equivalent to `concept.getAvailableAmalgamFcts().map(AmalgamationFct::getName)`